hail_scripts/reference_data/combine_test.py

```diff
@@ -125,17 +131,21 @@ def test_custom_select(self, mock_read_table):
                         MutationTaster_pred_id=3,
                         FATHMM_pred_id=1,
                         fathmm_MKL_coding_pred_id=1,
+                        GERP_RS=hl.eval(hl.float32(2.73)),
+                        phastCons100way_vertebrate=hl.eval(hl.float32(0.008)),
                     ),
                 ),
                 hl.Struct(
                     id=1,
                     mock_dbnsfp=hl.Struct(
-                        REVEL_score=0.052,
+                        REVEL_score=hl.eval(hl.float32(0.052)),
```

Comment:
ah does hail default to float64s?

Reply:
ah yes, great question.  yeah, hail defaults to float64s.  The idea here is to make the float32 exactly the same way hail is making it, so that the long tail of the float is equal.  [`assertAlmostEqual`](https://www.geeksforgeeks.org/python-unittest-assertalmostequal-function/) would be helpful here, but we're doing out assertions at the struct level and I didn't want to refactor the code here.